packages/js/product-editor/src/blocks/product-fields/downloads/edit.tsx

```diff
@@ -160,29 +105,43 @@ export function Edit( {
 		};
 	}
 
+	function handleUploadError( error: unknown ) {
+		createErrorNotice(
+			typeof error === 'string'
+				? error
+				: __( 'There was an error uploading files', 'woocommerce' )
```

Comment:
Could we add more context here?
For example, I tried to upload a file above 10mb ( which I think is the limit ) and it failed. But I had no way of knowing why it failed.
Especially errors around file size or unsupported file types would be good to know.

Reply:
Well, first that was not part of the ACs. Second, the error messages are not defined from designers yet. Third, we don't have a tune error handling for that yet in any of the upload components that we have. This should be covered in a specific issue about that and implemented in all similar components that we have.